### PR TITLE
ODIS V1 Sunset

### LINKED ISSUE #10352: Sunset ODIS v1
@alecps should know exactly what needs to happen here. Ideally todo with another person on the primitives team, to share the knowledge. 

This should happen within the next ~2 months ideally.

### PR DESCRIPTION
**Pull request by soloseng**
### Description

Removed all legacy code related to ODIS V1

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### Related issues

- Fixes #10352

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### Documentation

_The set of community facing docs that have been added/modified because of this change_

### CODE REVIEW

File: packages/phone-number-privacy/common/test/utils/input-validation.test.ts [hunk 1]
```diff
@@ -1,31 +1,7 @@
...
-        account: 'account',
-        hashedPhoneNumber: 'x'.repeat(REASONABLE_BODY_CHAR_LIMIT * 2),
-      }
-
-      const result = utils.isBodyReasonablySized(sampleData)
-
-      expect(result).toBeFalsy()
-    })
-  })
+  describe('isBodyReasonablySized utility', () => {})
```
Comment (soloseng) [inline packages/phone-number-privacy/common/test/utils/input-validation.test.ts]: @alecps Seems this test was only performed on `LegacyPnpQuotaRequest`. Is there any value in testing the body size for `PnpQuotaRequest`?

File: packages/phone-number-privacy/common/test/utils/input-validation.test.ts [hunk 2]
```diff
@@ -59,53 +35,5 @@ describe('Input Validation test suite', () => {
...
-        blindedQueryPhoneNumber: '',
-        account: 'acc',
-      }
-
-      const result = utils.hasValidBlindedPhoneNumberParam(sampleData)
-
-      expect(result).toBeFalsy()
-    })
-  })
+  describe('hasValidBlindedPhoneNumberParam utility', () => {})
```
Comment (soloseng) [inline packages/phone-number-privacy/common/test/utils/input-validation.test.ts]: ditto

File: packages/phone-number-privacy/signer/src/common/database/database.ts [hunk 3]
```diff
@@ -1,14 +1,8 @@
...
 import { Knex, knex } from 'knex'
 import { DEV_MODE, SignerConfig, SupportedDatabase, VERBOSE_DB_LOGGING } from '../../config'
-import { ACCOUNTS_COLUMNS, ACCOUNTS_TABLE } from './models/account'
 
-export async function initDatabase(
-  config: SignerConfig,
-  migrationsPath?: string,
-  doTestQuery = true
-): Promise<Knex> {
+export async function initDatabase(config: SignerConfig, migrationsPath?: string): Promise<Knex> {
```
Comment (soloseng) [inline packages/phone-number-privacy/signer/src/common/database/database.ts]: removed the `test` arg, as it was only used for legacy account tables

File: packages/phone-number-privacy/common/test/utils/input-validation.test.ts (hunk 1 above)
Comment (alecps) [reply packages/phone-number-privacy/common/test/utils/input-validation.test.ts]: We don't have to. Would be fine to delete this too

**Review by alecps — APPROVED** [review]
Looks great, thanks!!

**Comment by alecps** [comment]
Created this ticket to track dropping the Accounts table https://github.com/orgs/celo-org/projects/171/views/1?filterQuery=ODIS&pane=issue&itemId=34776697

File: packages/sdk/identity/src/odis/phone-number-identifier.ts [hunk 4]
```diff
@@ -39,8 +38,8 @@ export async function getPhoneNumberIdentifier(
   clientVersion?: string,
   blsBlindingClient?: BlsBlindingClient,
   sessionID?: string,
-  keyVersion?: number,
-  endpoint?: CombinerEndpointPNP.PNP_SIGN
+  keyVersion?: number
+  // endpoint?: CombinerEndpointPNP.PNP_SIGN
```
Comment (alecps) [inline packages/sdk/identity/src/odis/phone-number-identifier.ts]: flagging that this is just commented out

**Pull request merged**